docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
     <link rel="stylesheet" href="../../css/docs.css">
+    <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/docs_mobile.css">
     <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
   </head>
   <body>
-    <h1 class="heading-1">JWT Authentication</h1>
+      <main>
+      <h1 class="heading-1">JSON Web Token Authentication</h1>
+      <p class="block-text">A JSON Web Token (JWT) defines a compact and self-contained way for securely transmitting information between parties as a JSON object. You can find out more about JWTs at `JWT.io`.</p>
+      <p class="block-text">This guide uses JWT with Swift-JWT to implement Single Sign On (SSO) authentication for your Kitura routes.</p>
+      <h2 class="heading-2">Prerequisites</h2>
+      <ul class="plain-list">
+        <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide</li>
+        <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Kitura Style Guide:</a> Follow our guide to see how a production ready server should be structed</li>
+        <li><a onclick="localStorage.setItem('package', 'Swift-JWT'); setActiveSidebarElementForParent('swift-jwt')" href="../getting-started/update-package.html#">Update Dependencies:</a> Swift-JWT needs adding to the Package.swift file.</li>
+      </ul>
+      <div class="underline"></div>
+      <h2 class="heading-2">JWT Authentication</h2>
+      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create a JWTRoutes.swift file</h2>
+      <p>Open your `Sources` > `Application` > `Application.swift`</p>
+      <p> Inside the `postInit()` function add: </p>
+      <pre><code class="language-swift">initializeJWTRoutes(app: self)</code></pre>
+      <p>Create a new file called `JWTRoutes.swift` in `Sources` > `Application` > `Routes`</p>
+      <p>Inside this file, add the framework for our routes code:</p>
+      <pre><code class="language-swift">import Foundation
+import KituraContracts
+import SwiftJWT
+
+func initializeJWTRoutes(app: App) {
+    app.router.post("/jwtlogin") { request, response, next in
+        // Read credentials and generate JWT here
+        next()
+    }
+
+    app.router.get("/jwtprotected") { request, response, next in
+        // Verify JWT here
+        next()
+    }
+}
+extension App {
+  // Define JWT signer and verifier here
+}</code></pre>
     <div class="underline"></div>
-    <h2 class="heading-2">Next steps</h2>
-    <p><span class="blue-text bold">Add logging:</span> Get useful feedback from your server about startup and errors</p>
-    <p><span class="blue-text bold">Add routing:</span> Add REST APIs, such as HTTP GET, to your server</p>
+    <h2 class="heading-2"><span class="blue-text">Step 2:</span> Define your JWTSigner and JWTVerifier</h2>
```

Comment:
Define your JWTSigner and JWTVerifier didn't make sense to me, can this be more generic something like... "Create and sign the JWT instance" or whatever it is we're doing?

We also need to have a comment under the title stating that we support 3 methods of whatever it is we're doing in as I wondered what on earth the buttons were for (until I twigged what was going on!).